NLog.Web.Tests/LayoutRenderers/AspNetSessionValueLayoutRendererTests.cs

```diff
@@ -23,6 +24,7 @@ public AspNetSessionValueLayoutRendererTests()
 
         public void SetUp()
         {
+            ConfigurationItemFactory.Default.LayoutRenderers.RegisterDefinition("aspnet-session", typeof(AspNetSessionValueLayoutRenderer));
```

Comment:
I used both the resharper xunit test runner and the ncrunch test runner, and without that, I got 
```
System.ArgumentException: LayoutRenderer cannot be found: 'aspnet-session'
   at NLog.Config.Factory`2.CreateInstance(String name)
   at NLog.Layouts.LayoutParser.ParseLayoutRenderer(ConfigurationItemFactory configurationItemFactory, SimpleStringReader sr)
   at NLog.Layouts.LayoutParser.CompileLayout(ConfigurationItemFactory configurationItemFactory, SimpleStringReader sr, Boolean isNested, String& text)
   at NLog.Layouts.SimpleLayout.set_Text(String value)
   at NLog.Layouts.Layout.op_Implicit(String text)
   at NLog.Web.Tests.LayoutRenderers.AspNetSessionValueLayoutRendererTests.SessionWithCulture()
```

the packages.config shows NLog 4.0.1.